source/support.tex

```diff
@@ -3951,10 +3951,10 @@
 from one of the following exposition-only enumerations:
 
 \begin{codeblock}
-enum class eq { equal = 0, equivalent = equal,
-                nonequal = 1, nonequivalent = nonequal };   // \expos
-enum class ord { less = -1, greater = 1 };                  // \expos
-enum class ncmp { unordered = -127 };                       // \expos
+enum class @\placeholder{eq}@ { @\placeholder{equal}@ = 0, @\placeholder{equivalent}@ = @\placeholder{equal}@,
```

Comment:
Should we use the non-correcting forms of placeholder here to maintain vertical alignment? Can you maybe attach a diff screenshot?

Reply:
Good idea - the comments are currently misaligned. I'll `nc` these.

Reply:
Fixed with a "fixup" commit.